### PR TITLE
cleanup: removing all the material for the old site

### PR DESCRIPTION
**Pull request by dpsmith**
In preparation for the upcoming site update, cleaning out all the old files.

### CODE REVIEW

**Review by miczyg1 — APPROVED** [review]
Just a side note, would it be worth to leave CONTRIBUTING.md with an info how to contribute to the new mkdocs documentation? Probably subject for a new PR.

File: README.md [hunk 1]
```diff
@@ -1,68 +1,6 @@
...
-
-Start Bootstrap is based on the [Bootstrap](http://getbootstrap.com/) framework created by [Mark Otto](https://twitter.com/mdo) and [Jacob Thorton](https://twitter.com/fat).
-
-## Copyright and License
-
-Copyright 2013-2019 Blackrock Digital LLC. Code released under the [MIT](https://github.com/BlackrockDigital/startbootstrap-grayscale/blob/gh-pages/LICENSE) license.
+The TrenchBoot website has been converted over to
+[Material for MkDocs](https://squidfunk.github.io/mkdocs-material/). The
+content for the original site is still available under the
+[old-site-archive](https://github.com/TrenchBoot/TrenchBoot.github.io/tree/old-site-archive).
```
Comment (pietrushnic) [inline README.md:6]: @dpsmith what do you think about rules we have in Dasharo docs readme? https://github.com/Dasharo/docs#readme

Comment (pietrushnic) [reply README.md:6]: @dpsmith ?

Comment (dpsmith) [reply README.md:6]: i'm good with this.

**Pull request merged**